iib/web/iib_static_types.py

```diff
@@ -109,6 +110,19 @@ class RmRequestPayload(TypedDict):
     overwrite_from_index_token: Optional[str]
 
 
+class FbcOperationRequestPayload(TypedDict):
+    """Datastructure of the request to /builds/fbc-operation API point."""
+
+    fbc_fragment: str
+    from_index: str
+    binary_image: NotRequired[str]
+    build_tags: NotRequired[List[str]]
+    add_arches: NotRequired[List[str]]
+    overwrite_from_index: NotRequired[bool]
+    overwrite_from_index_token: NotRequired[str]
+    batch: NotRequired[str]
```

Comment:
Do we need distribution_scope here?

Reply:
Should not it be the same as in from_index? So this should not be needed to specify in request?

Reply:
Hey Jan
bringing this question again, how do we decide when do we take distribution_scope in the request and when we don't? I see in the api_v1.py file that the field is accepted in request payload for other requests like add, rm , merge?